django-stubs/db/models/fields/files.pyi

```diff
@@ -33,6 +34,8 @@ class FieldFile(File):
     def closed(self) -> bool: ...
     def __getstate__(self) -> dict[str, Any]: ...
     def __setstate__(self, state: dict[str, Any]) -> None: ...
+    def __eq__(self, other: FieldFile | str) -> bool: ...
```

Comment:
It can be `object`, because any value (with correct `__eq__`) can be compared with `self.name`